Database covariate storage

azimov opened the pull request:
Resolves PR #152 , storing results in DB rather than Andromeda object. Also fixes likely errors that occured before and allows specification to store `timeRef` and `covariatesContinuous` in database tables with newly specified `targetTables` list parameter.

azimov committed:
Made commented sqlite test for old code work

azimov committed:
Changes to interface of getDefaultCovariates

azimov committed:
Fixes, restoration of old parameters and adaptation to use temp tables

azimov committed:
Creating/dropping or appending results checked. Added details to news

azimov committed:
whitespace

anthonysena committed:
Merge branch 'develop' into database-covariate-storage

anthonysena commented:
@azimov - could you merge in the latest code on `develop` to this branch and resolve any conflicts? Then @ginberg and I will take a look. Thanks!

anthonysena committed:
Merge branch 'develop' into database-covariate-storage

anthonysena committed:
Fix merge problems

anthonysena committed:
Remove debugging message

anthonysena committed:
Merge branch 'develop' into database-covariate-storage

anthonysena committed:
Restore warning for post-coordinated concept collision

jreps committed:
fixing export FE to database

- letting users export to a set of database tables instead of downloading to Andromeda

jreps committed:
Update CreateExportTables.sql

fixing time_ref columns

jreps committed:
changing temp table insert

jreps committed:
Update GetDefaultCovariates.R

jreps committed:
Merge branch 'database_covariate_storage_jenna' into database-covariate-storage

jreps committed:
Update DemographicsTime.sql

explicitly specifying where the observation_period_ columns come from just in case the cohort table has these are columns

jreps committed:
Update DemographicsTime.sql

missed period_id

jreps committed:
Update GetDefaultCovariates.R

fixing oracleTempSchema being used

anthonysena committed:
Update roxygen2 comments, unit tests and fix errors (#322)

* Updates to remove unnecessary tests, add back collision detection and to properly pass off parameters to covariate builder functions

* Update docs

* Fix parameter documentation and example

anthonysena committed:
Bump version and date in description and update news.md
